training/backends/kubernetes.js

```diff
@@ -117,15 +131,19 @@ class KubernetesTaskRunner {
     }
 }
 
+
+
 module.exports = async function execTask(job, spec) {
+    const jobName = Config.TRAINING_KUBERNETES_JOB_NAME_PREFIX + 'training-job-' + job.id + '-' + spec.name;
+    const jobIdTask = `${job.id}-${spec.name}`;
     const k8sJob = {
         apiVersion: 'batch/v1',
         kind: 'Job',
         metadata: {
-            name: Config.TRAINING_KUBERNETES_JOB_NAME_PREFIX + 'training-job-' + job.jobId + '-' + spec.name,
+            name: jobName,
             labels: {
                 app: 'training-job',
-                'edu.stanford.almond/job-id': job.id
+                'edu.stanford.almond/job-id-task': jobIdTask
```

Comment:
Took me a while to figure out that `job-id-task` is the combination of job id and task name. Maybe `task-id` is more readable? Or maybe two different labels?